README.md

```diff
@@ -78,18 +78,21 @@ Sample help commandlines:
 ```
 ```
     $ jit-diff diff --help
-    usage: jit-diff diff [-b <arg>] [-d <arg>] [-o <arg>] [-a] [-t <arg>]
-                    [-m] [-f] [-v] [--core_root <arg>] [--test_root <arg>]
-
-        -b, --base <arg>        The base compiler exe or tag.
-        -d, --diff <arg>        The diff compiler exe or tag.
+    usage: jit-diff diff [-b <arg>] [-d <arg>] [--crossgen <arg>] [-o <arg>]
+                    [-a] [-t <arg>] [-c] [-f] [--benchmarksonly] [-v]
+                    [--core_root <arg>] [--test_root <arg>]
+    
+        -b, --base <arg>        The base compiler path or tag.
+        -d, --diff <arg>        The diff compiler path or tag.
```

Comment:
I added more details. Note that I've changed the base and diff to be the directories containing crossgen/clrjit, which is consistent with the config files.